static/sass/_patterns_media-object--snap.scss

```diff
@@ -1,109 +1,107 @@
 // Based on the matrix component https://github.com/vanilla-framework/vanilla-framework/blob/develop/scss/_patterns_matrix.scss
 @mixin snapcraft-p-media-object--snap {
-  $matrix-img-width: map-get($icon-sizes, thumb--small);
-  $matrix-img-gutter: $sph-inter;
-
-  .p-media-object--snap {
-    display: block;
-    margin-bottom: $spv-inter--regular * 2;
-
-    &__img {
-      align-self: flex-start;
-      height: $matrix-img-width;
-      margin-bottom: $spv-inter--condensed;
-      margin-right: $matrix-img-gutter;
-      max-height: $matrix-img-width;
-      max-width: $matrix-img-width;
-      width: auto;
-    }
-
-    &__content {
-      display: flex;
-      flex: 1 1 auto;
-      flex-direction: column;
-      padding-right: $sph-intra;
+  %media-object {
+    .p-media-object__details {
```

Comment:
Does it have anything to do with Vanilla `p-media-object` (https://docs.vanillaframework.io/en/patterns/media-object)?
It seems to use and override same class names, not sure if that's intentional or not...

Reply:
yeah, it expects `.p-media-object` as a base and modifies a few things if the classname has `p-media-object p-media-object--snap`

I changed this from how it was before because it repeated a lot of the `p-media-object` class and used different names within.